### PR TITLE
Updating google-chrome-stable package name; using dnf instead of yum.

### PR DESCRIPTION
**Pull request by crobby**

### CODE REVIEW

File: tests/Dockerfile [hunk 1]
```diff
@@ -11,11 +11,11 @@ RUN dnf install -y bc git go-toolset python3-pip unzip && \
     git clone https://github.com/crobby/peak $HOME/peak && \    
     cd $HOME/peak && \
     git submodule update --init
-
-RUN yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm &&\
+RUN echo "Installing chrome"
```
Comment (anishasthana) [inline tests/Dockerfile:14]: Do you still need this echo here?

Comment (crobby) [reply tests/Dockerfile:14]: doh

Comment (crobby) [reply tests/Dockerfile]: just making sure that someone was paying attention this morning...good work

**Review by anishasthana — APPROVED** [review]
lgtm

**Review by LaVLaS — APPROVED** [review]
/lgtm

**Pull request merged**